packages/js/client/src/__tests__/e2e/workflow-test-cases.ts

```diff
@@ -1,81 +1,189 @@
-import { Workflow } from "@polywrap/core-js";
+import { JobResult, JobStatus, MaybeAsync, Workflow } from "@polywrap/core-js";
+import { GetPathToTestWrappers } from "@polywrap/test-cases";
+import path from "path";
 
-export const sanityWorkflow: Workflow = {
-  name: "simple-storage",
-  jobs: {
-    cases: {
-      steps: [
-        {
-          uri: "ens/testnet/simple-storage.eth",
-          method: "deployContract",
-          args: {
-            connection: null,
-          },
+export interface WorkflowTestCase {
```

Comment:
Why do we have test cases as a separate file if we're only using them in one test file?

Reply:
Just for more readability